### PR TITLE
[1.0] Allow testbench 5 for laravel 7

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,15 +15,15 @@
     ],
     "require": {
         "php": "^7.1.3",
-        "illuminate/support": "^5.8|^6|^7",
+        "illuminate/support": "^5.8|^6.0|^7.0",
```
Comment (barryvdh) [inline composer.json]: This shouldn't make a difference, right? Is this just style preference?

**Comment by GrahamCampbell** [comment]
Is there going to be an upstream release, so we can update the tests to expect 204 instead of 200. so they can pass on all versions?

**Comment by GrahamCampbell** [comment]
@barryvdh

**Comment by barryvdh** [comment]
Tagged 1.3.0, so we can update the constraints/tests.

**Pull request merged**